docs/specs/tripgo.swagger.yaml

```diff
@@ -3293,6 +3293,17 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/NearbyCarParkLocation'
+      externalData:
+        type: object
+        description: It is a dictionary from "operatorName" and specific inforamtion from that operator.
```

Comment:
Typo in "information" and maybe better to rename "operatorName" to "sourceName" as it could be from any source, not just an operator.

Reply:
Code updated. Renamed and fixed typo fixed.